src/app/components/containers/flow-container/sankey/sankey.component.spec.ts

```diff
@@ -21,7 +21,7 @@ import { SankeyComponent } from './sankey.component';
 class TestSankeyComponent {
   @ViewChild(SankeyComponent) child: SankeyComponent;
   sankeyData = Mocks.Sankey;
-  selectNode(data): void {}
+  selectNode(data): void { }
```

Comment:
Nitpick: Unnecessary space added here (possibly done automatically by a code style tool?)

Reply:
yeah, those where introduced when formatting the files with vscode.

Reply:
I figured it must be auto-formatting. Looks like the standard VS Code language settings add the space here, I use Prettier which removes them.